File: DataServer/Models/Table.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace DataServer.Models
{
    public class Table
    {
        public int TableId { get; set; }
        public int Capacity { get; set; }
        [JsonIgnore]
        public List<TableBooking> TableBookings { get; set; }
    }
}

File: RestaurantServer/src/main/java/dk/restaurant/models/Table.java
package dk.restaurant.models;

import java.util.List;

public class Table
{
  private int tableId;
  private int capacity;
  private List<TableBooking> tableBookings;
  public Table(int tableId, int capacity, List<TableBooking> tableBookings)
  {
    this.tableId = tableId;
    this.capacity = capacity;
    this.tableBookings = tableBookings;
  }

  public Table() {}

  public int getTableId()
  {
    return tableId;
  }

  public void setTableId(int tableId)
  {
    this.tableId = tableId;
  }

  public int getCapacity()
  {
    return capacity;
  }

  public void setCapacity(int capacity)
  {
    this.capacity = capacity;
  }

  public List<TableBooking> getTableBookings() {
    return tableBookings;
  }

  public void setTableBookings(List<TableBooking> tableBookings) {
    this.tableBookings = tableBookings;
  }
}
